genie-core/src/test/java/com/netflix/genie/core/configs/ServicesConfigTest.java

```diff
@@ -408,4 +417,26 @@ public FactoryBean fileTransferFactory() {
     public JobsProperties jobsProperties() {
         return new JobsProperties();
     }
+
+    /**
+     * Returns a temporary directory as the jobs resource.
+     *
+     * @return The job dir as a resource.
+     * @throws IOException If there is a problem.
+     */
+    @Bean
+    public Resource jobsDir() throws IOException {
+        final File jobsDir = Files.createTempDir();
```

Comment:
A) I think we already have this bean somewhere
B) please use the `Junit` rule that creates a temporary directory/ file for these things. That guarantees its cleaned up at the end of tests